scripts/EN/Cybil

```diff
@@ -1846,16 +1846,16 @@ LoadBackground(0x191); // 0x24f8: 00001069
 DrawBackground(); // 0x24fe: 00000065
 OP_0x29(); // 0x2502: 00000029
 PlaySecondSoundFx(0x28); // 0x2506: 00001073
-ShowText((5237)"She kicks the door with all her might. With a hopeful feeling, she continues to kick the door over and over with the thick sole of her boot. Finally the knob breaks, and the door forcefully opens inward. At the same time hot wind singes Cybil's hair."); // 0x250c: 0000100c
+ShowText((5237)"She kicks the door with all her might.¥nW2With a hopeful feeling, W1she continues to kick the door over and over with the thick sole of her boot.¥n¥nW2Finally the knob breaks, W1and the door forcefully opens inward. At the same time hot wind singes Cybil's hair."); // 0x250c: 0000100c
 HandleInput(); // 0x2512: 00000010
 CleanScreen(); // 0x2516: 00000064
 LoadBackground(0x192); // 0x251a: 00001069
 LoadEffect(0x2e, 0x1, 0x1); // 0x2520: 0000306d
 DrawBackground(); // 0x252a: 00000065
 PlaySoundFx(0x26); // 0x252e: 00001072
-ShowText((5238)"And then Cybil saw it. The room is about as large as a movie theater. Fire has spread all throughout, and it has began to collapse."); // 0x2534: 0000100c
+ShowText((5238)"And then Cybil saw ¥nW2 a room about as large as a movie theater. W1Flames are rising up through this room, W1and its walls have begun to collapse."); // 0x2534: 0000100c
```

Comment:
Fairly big changes to include a missing bit of Japanese.

For style purposes I also combined the first two sentences.